collector/meminfo_numa_linux.go

```diff
@@ -106,8 +106,14 @@ func getMemInfoNuma() ([]meminfoMetric, error) {
 		}
 		defer numastatFile.Close()
 
+		vmstatFile, err := os.Open(filepath.Join(node, "vmstat"))
+		if err != nil {
+			return nil, err
+		}
+		defer vmstatFile.Close()
+
 		nodeNumber := meminfoNodeRE.FindStringSubmatch(node)
-		if nodeNumber == nil {
+		if nodeNumber == nil || len(nodeNumber) != 2 {
```

Comment:
Maybe you can put that number into constant? Something like `expectedNodeNumberLength`.